index.html

```diff
@@ -2,241 +2,277 @@
-            </a>
-          </li>
-          <li class="social-media-icon">
-            <a href="https://www.linkedin.com/in/isaac-imaobong-samuel-a4849b1b8/" target="_blank"
-              rel="noopener noreferrer">
-              <img class="s-m-icon" src="Image/Linkedin.svg" alt="Linkedin">
-            </a>
-          </li>
-          <li class="social-media-icon">
-            <a href="#social-media-icon" target="_blank" rel="noopener noreferrer">
-              <img class="s-m-icon" src="Image/Blog.svg" alt="Blog">
-            </a>
-          </li>
-          <li class="social-media-icon">
-            <a href="https://github.com/zieeco" target="_blank" rel="noopener noreferrer">
-              <img class="s-m-icon" src="Image/github.svg" alt="github">
-            </a>
-          </li>
-          <li class="social-media-icon">
-            <a href="#social-media-icon" target="_blank" rel="noopener noreferrer">
-              <img class="s-m-icon" src="Image/angelist.svg" alt="Angelist">
-            </a>
-          </li>
-        </ul>
-        <button class="about__myself__btn btn" type="button">Get my resume</button>
-        <ul class="skill__section">
-          <li class="skill__section__container">
-            <div class="skill__info">
-              <h2 class="skill__title">Languages</h2>
-              <img src="Image/arrowdown.svg" alt="arrow-dropdowm">
-            </div>
-            <ul class="language__list">
-              <li class="language">
-                <img src="Image/JS.svg" alt="javascript">
-                <h2 class="language__name">JavaScript</h2>
-              </li>
-              <li class="language">
-                <img src="Image/HTML.svg" alt="Html">
-                <h2 class="language__name">HTML</h2>
-              </li>
-              <li class="language">
-                <img src="Image/CSS.svg" alt="Css">
-                <h2 class="language__name">CSS</h2>
-              </li>
-            </ul>
-          </li>
-          <li class="skill__info">
-            <h2 class="skill__title">Framework</h2>
-            <img src="Image/arrowright.svg" alt="arrow-dropdowm">
-          </li>
-          <li class="line"></li>
-          <li class="skill__info">
-            <h2 class="skill__title">Skills</h2>
-            <img src="Image/arrowright.svg" alt="arrow-dropdowm">
-          </li>
-          <li class="line"></li>
-        </ul>
-      </section>
-    
+        </li>
+        <li class="skill__info">
+          <h2 class="skill__title">Framework</h2>
+          <img src="Image/arrowright.svg" alt="arrow-dropdowm">
+        </li>
+        <li class="line"></li>
+        <li class="skill__info">
+          <h2 class="skill__title">Skills</h2>
+          <img src="Image/arrowright.svg" alt="arrow-dropdowm">
+        </li>
+        <li class="line"></li>
+      </ul>
+    </section>
+  </article>
+
+  <!-- Footer -->
+  <footer class="footer">
```

Comment:
- [ ] According to the Figma design this section should have a background image. I think it's the same issue with the `SOURCE_PATH` for the background because you are using the `/` before. I would suggest updating this in order to match the Figma design :raised_hands:

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/139756143-56e5134a-9e14-4a79-ba2b-dbb6b6af22ff.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/139756189-558fd331-94b8-4416-8ffd-742c5243aa35.png)